aiida_pseudo/cli/family.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""Commands to inspect or modify the contents of pseudo potential families."""
+import json
+
+import click
+
+from aiida.cmdline.utils import decorators, echo
+
+from ..groups.mixins import RecommendedCutoffMixin
+from .params import arguments, options
+from .root import cmd_root
+
+
+@cmd_root.group('family')
+def cmd_family():
+    """Command group to inspect or modify the contents of pseudo potential families."""
+
+
+@cmd_family.group('cutoffs')
+def cmd_family_cutoffs():
+    """Command group to inspect or modify the recommended cutoffs of pseudo potential families."""
+
+
+@cmd_family_cutoffs.command('set')
+@arguments.PSEUDO_POTENTIAL_FAMILY()
+@click.argument('cutoffs', type=click.File(mode='rb'))
```

Comment:
Is there a reason for making the cutoffs an argument and the stringency a required option?

Reply:
Hmm, in the current conception. I think originally I intended to allow to specify multiple stringencies at once and then let one define the default. But indeed, now that only on stringency is defined at a time, maybe it makes more sense to specify it as an argument. On the other hand, leaving it like this does make it possible in the future to make the command more flexible and also allow a file with multiple stringencies defined, where the keys are the stringency name. Then it makes more sense to make the default stringency parameter an option and use the first one defined in the file as the default such that the flag becomes optional. Think that is the best option actually thinking about it now